tools/presubmit-pre-push-hook

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
why do we need this bash proxy script? can't we just set tools/run_presubmit in the hook and mark that file executable with a python3 shebang?

Reply:
Done.